src/components/ha-icon-picker.ts

```diff
@@ -46,9 +85,12 @@ export class HaIconPicker extends LitElement {
 
   @property({ type: Boolean }) public invalid = false;
 
-  @state() private _opened = false;
+  @state() private _filterString = "";
 
-  @query("ha-combo-box", true) private comboBox!: HaComboBox;
+  protected override async scheduleUpdate() {
+    await nextRender();
+    super.scheduleUpdate();
+  }
```

Comment:
We should add comment to explain why we do this.

Reply:
Will do.